src/PrestaShopBundle/Api/QueryParamsCollection.php

```diff
@@ -657,6 +657,10 @@ protected function appendSqlSearchFilter($filters)
             return $filters;
         }
 
+        if (!is_array($this->queryParams['filter']['keywords'])) {
+            $this->queryParams['filter']['keywords'] = (array)$this->queryParams['filter']['keywords'];
```

Comment:
It's generally best to handle this case like this:
```php
$this->queryParams['filter']['keywords'] = array($this->queryParams['filter']['keywords']);
```
Casting an object into array will transform it into a hash-map: https://3v4l.org/ihISI

Reply:
Yeah this is what I wanted to do. Do you need a dump of the current value?

Reply:
If it's a hash-map you wanted, it's fine by me 👍